simpleocr/files.py

```diff
@@ -45,45 +32,152 @@ def write(self):
         write_boxfile(self.path, self.classes, self.segments)
 
 
-class ImageFile(object):
```

Comment:
I took a quick look, and it looks good :) Feel free to push it here
A quick thought: why use `Imagebuffer` and `GroundBuffer` vs `Image` and `Ground`?
The `GroundBuffer` doesn't really contain a buffer, and even on Image, it's mostly a implementation detail that is subject to change (say, we can store PIL objects in the future).